src/config/client_config.h

```diff
@@ -71,31 +71,31 @@ class ClientConfigList {
 /// \brief Provides information about one manual client.
 class ClientConfig {
 public:
-    ClientConfig();
+    ClientConfig() = default;
 
     /// \brief Creates a new ClientConfig object.
     /// \param flags quirks flags
     /// \param ip ip address
     /// \param userAgent user agent
     ClientConfig(int flags, std::string_view ip, std::string_view userAgent);
 
-    const std::unique_ptr<ClientInfo>& getClientInfo() const { return clientInfo; }
+    const ClientInfo& getClientInfo() const { return clientInfo; }
```

Comment:
This can become critical if `ClientConfig` instance is destroyed before the referenced object.

Reply:
How to test that?